PythonToHash

```diff
@@ -0,0 +1,27 @@
+# Python rogram to find the SHA-1 message digest of a file
```

Comment:
Python program to find the SHA-1 message digest of a file( you missed a word `p` in program
And keep this comment below imports for better readability